### PR TITLE
WIP: Update file handling from n-quads to n-triples

### LINKED ISSUE #28: NTriples Support
We need support for writing out NTriples to the Git repository.

This will also make #32 obsolete.

### PR DESCRIPTION
**Pull request by splattater**
- [x] Update testrepo factory and application tests from nq to nt.
- [x] Update configuration, core and changeset application (helpers) to handle nt files instead of nq files.
- [x] Update documentation
- [x] Merge/Rebase with origin/master since there is a lot of new stuff (git.py, merger and dependencies with test_git.py) (maybe we should do this together @white-gecko )
- [x] Fix merge: @white-gecko 

- Removed graph config mode

Fix #28.

### CODE REVIEW

**Comment by splattater** [comment]
this will fix #28

**Comment by white-gecko** [comment]
Do you have any idea, what we want to do about the test coverage?

File: quit/core.py [hunk 1]
```diff
@@ -324,58 +323,50 @@ def changeset(self, commit):
...
                 except KeyError:
-                    tmp = ConjunctiveGraph()
-                    tmp.parse(data=entity.content, format='nquads')
-
-                    # Info: currently filter graphs from file that were not defined in config
-                    # Todo: is this the wanted behaviour?
-                    contexts = set((context for context in tmp.contexts(None)
-                                    if context.identifier in map)) | graphsFromConfig
+                    tmp = Graph(identifier=graphUri)
+                    tmp.parse(data=entity.content, format='nt')
```
Comment (white-gecko) [inline quit/core.py]: Is is possible to avoid variable names like `tmp`?

File: quit/core.py [hunk 2]
```diff
@@ -411,19 +402,16 @@ def getFileReferenceAndContext(self, blob, commit):
         if commit.id not in self._graphconfigs:
             self.updateGraphConfig(commit.id)
 
-        uriFileMap = self._graphconfigs.get(commit.id).getgraphurifilemap()
+        # uriFileMap = self._graphconfigs.get(commit.id).getgraphurifilemap()
```
Comment (white-gecko) [inline quit/core.py]: Can this out-comment line be removed?

**Pull request merged**